.changeset/clean-tools-double.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/checkbox": major
```

Comment:
```suggestion
"@nextui-org/checkbox": patch
```

Reply:
Hi [jrgarciadev](https://github.com/jrgarciadev) ,
I resolved this
[jrgarciadev](https://github.com/jrgarciadev)